### ISSUE TITLE
since upgrade I've lost cron job

### ISSUE THREAD

**Issue by prysme01** [YunoHost-Apps/freshrss_ynh#77]
Since the latest upgrade, my feed are not updated automatically anymore.

How can I check where is supposed to be the cron job ? freshrss user do not have a shell so no cron.

**Comment by plopoyop** [comment]
hey,
duplicated from https://github.com/YunoHost-Apps/freshrss_ynh/issues/76

please remove /tmp/FreshRSS.log

**Comment by prysme01** [comment]
Ok done, but where can I find the cron job ?

**Comment by plopoyop** [comment]
the cron is installed into '/etc/cron.d/freshrss'
is your problem fixed by removing the log file ?

**Comment by roukydesbois** [comment]
Hello all,

I faced the same problem, and removing the /tmp/freshrss.log file didn't solve the issue. Inside the log file, there was a mention that some packages (curl and ok) were missing, and I had the same error displyed when I ran the command of the cron job with the proper user.
I fixed the issue by changing in the cron job the php binary used from php7.0 to php - some packages, curl and ok I guess, are now with php and not php7.0.
